camel/toolkits/search_toolkit.py

```diff
@@ -947,6 +947,151 @@ def search_bing(self, query: str, max_results: int = 5) -> Dict[str, Any]:
         except Exception as e:
             return {"error": f"Bing scraping error: {e!s}"}
 
+    @api_keys_required([(None, 'EXA_API_KEY')])
+    def search_exa(
+        self,
+        query: str,
+        search_type: Literal["auto", "neural", "keyword"] = "auto",
+        category: Optional[
+            Literal[
+                "company",
+                "research paper",
+                "news",
+                "pdf",
+                "github",
+                "tweet",
+                "personal site",
+                "linkedin profile",
+                "financial report",
+            ]
+        ] = None,
+        num_results: int = 10,
+        include_text: Optional[List[str]] = None,
+        exclude_text: Optional[List[str]] = None,
+        use_autoprompt: bool = True,
+        text: bool = False,
+    ) -> Dict[str, Any]:
+        r"""Use Exa search API to perform intelligent web search with optional
+        content extraction.
+
+
```

Comment:
remove additional line
```suggestion
        r"""Use Exa search API to perform intelligent web search with optional
        content extraction.

```